.github/workflows/build_pr.yaml

```diff
@@ -0,0 +1,82 @@
+name: Build PR
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [ opened, synchronize, reopened, ready_for_review ]
+
+env:
+  BASE_PATH: /familie/kontantstotte/soknad/
+
+jobs:
+  test:
+    name: Kjør typecheck og tester
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          registry-url: "https://npm.pkg.github.com"
+      - name: Installer node-pakker
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.READER_TOKEN }}
+        run: yarn --prefer-offline --frozen-lockfile
+      - name: Kjør typecheck
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.READER_TOKEN }}
+        run: yarn typecheck
+      - name: Kjør tester
+        env:
+          CI: true
+          NODE_AUTH_TOKEN: ${{ secrets.READER_TOKEN }}
+        run: yarn test
+
+  bygg-yarn-for-dependabot:
```

Comment:
bygg-yarn-for-dependabot og Bygg app steget er helt likt hvis du fjerner den unødvendige ifen

Reply:
Ah, godt sett!

Fjerna dependabot steget og fjerna if'n.